app/jobs/perilune/tasks/executor_job.rb

```diff
@@ -72,15 +78,33 @@ def tracer
       end
 
       def track_stats(success:)
-        count_hash = success ? { count: 1, success: 1 } : { count: 1, failure: 1 }
-        inner_hash = { task.task_klass.downcase.intern => count_hash }
+        task_type = task.task_type.downcase == 'import' ? 'import' : 'export'
+        state = success ? 'success' : 'failure'
+
         Trifle::Stats.track(
-          key: "perilune::#{task.task_type.downcase}::#{task.task_klass.downcase}",
+          key: "perilune::#{task_type}::#{state}",
           at: Time.zone.now,
           config: Perilune.default.stats_driver_config,
-          values: count_hash.merge(inner_hash)
+          values: trifle_values_hash(success:, task_type:)
         )
       end
+
+      def duration_to_ms
+        return 0 unless @duration.present? && !@duration.zero?
+
+        (@duration * 1000).round
+      end
+
+      def trifle_values_hash(success:, task_type:)
+        @duration = @duration ? @duration * 1000 : 0
+
+        {
+          task_type.to_sym => {
+            count: 1,
+            duration: duration_to_ms
+          }
+        }
```

Comment:
Finally, the tracked object will be like this:
```ruby
Trifle::Stats.track(
  key: "perilune::import::failure", # or export/success
  at: DateTime.now,
  values: {
    :import => {
      count: 1,
      duration: 87 # in ms, rounded
    }
  }
)
```

With this structure, I can easily filter and search on:
- key: "perilune::import::success"
- key: "perilune::import::failure"
- key: "perilune::export::success"
- key: "perilune::export::failure"